resources/js/components/forms/photo/PhotoTagDialog.vue

```diff
@@ -97,6 +90,7 @@ function execute() {
 			life: 3000,
 		});
 		AlbumService.clearCache(props.parentId);
+		TagsService.clearCache();
```

Comment:
This is a general question as I get oriented in the codebase, not a problem with this PR:

How well do we expect the Lychee UI to scale as more albums and tags are added? I have noticed a number of spots where we effectively clear the cache then re-fetch _all_ albums/tags/etc, and I am wondering how well I should expect my Lychee instance's UI to scale.

Or, put another way: how many albums are in the largest known Lychee instances?

Reply:
> I have noticed a number of spots where we effectively clear the cache then re-fetch all albums/tags/etc, and I am wondering how well I should expect my Lychee instance's UI to scale.

Cache management is always a difficult thing. :| The implementation in Lychee is not perfect, but it aims to help.